README.md

```diff
@@ -0,0 +1,5 @@
+### LaunchDarkly Sample Javascript Application  ###
+We've built a simple console application that demonstrates how LaunchDarkly's SDK works.  Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/).
+##### Build instructions  #####
+1. Copy your feature flag key from your LaunchDarkly dashboard into the snippet in `index.html` 
```

Comment:
You also have to copy a JavaScript snippet into `index.html`. Take a look at the Quickstart instructions for more accurate text.